frontend/src/container/Tracim.jsx

```diff
@@ -145,31 +145,36 @@ export class Tracim extends React.Component {
     ])
   }
 
-  handleUserCallCreated = (tlm) => {
+  handleUserCallCreated = async (tlm) => {
     const { props } = this
     const bell = '🔔'
     const isMainTab = this.liveMessageManager.eventSource !== null
-    const notificationOptions = { tag: 'call', renotify: true, requireInteraction: true }
 
     if (tlm.fields.user_call.callee.user_id === props.user.userId) {
-      const notification = new Notification(tlm.fields.user_call.caller.public_name + props.t(' is calling you on Tracim'), notificationOptions)
+      if ('Notification' in window) {
+        const notificationString = tlm.fields.user_call.caller.public_name + props.t(' is calling you on Tracim')
```

Comment:
maybe put the name in the translation string as a placeholder. This with make it easier to translate

props.t('{name} is calling you on Tracim', ...)